### ISSUE TITLE
Tweak mutations colors for better readability

### ISSUE THREAD

**Issue by ivan-aksamentov** [hodcroftlab/covariants#26]
Some of the colors on badges in "Defining mutations" section have insufficient contrast ratio against text and/or background. Simply put, it's hard to read the text.

We want to change the colors, for better readability.

The colors are here
https://github.com/hodcroftlab/covariants/blob/master/web/src/colors.ts

Originally, the Nuc and AA colors are taken from bioSyntax project:
https://github.com/bioSyntax/bioSyntax
more specifically, their spreadsheet
https://github.com/bioSyntax/bioSyntax/blob/429671cfbf6d7355e1db25dc8116687dd8683029/dev/theme/bioSyntax_theme.xls

The gene colors in the same file are taken from Nextclade, which are taken from Auspice. These are the same as in the Auspice entropy panel.

That is, there are less colors in auspice than we need and auspice reuses the same colors in cycle. I ran this cycle for SARS-CoV-2 genes and made a harcoded dictionary. I also made gene S to be red and gene N to be blue, don't remember exactly, so that they are not of the same color.
https://github.com/nextstrain/nextclade/blob/1fbc1e924bd3f58e0681f0430d033e5dee83e776/packages/web/src/constants.ts#L31-L44

Additional points for making the colors colorbindness-friendly, but given how many colors there are, it's a tough task.

**Comment by emmahodcroft** [comment]
One idea posed by Ivan: make colours slightly more pastel so they are a bit less glaring. But will preserve seeing similar AA have similar properties :)

**Comment by emmahodcroft** [comment]
A nice colour scheme found by Richard: http://ugene.net/forum/YaBB.pl?num=1337064665 (would still perhaps need a little tweaking), with good explanations of the colours, too.

**Comment by ivan-aksamentov** [comment]
We could also add somewhere a legend that will explain colors, if that's useful.

**Comment by emmahodcroft** [comment]
I was thinking this might fit in well in the FAQ section perhaps?
